test/ui/pages/config/WidgetFrame.test.js

```diff
@@ -163,15 +163,15 @@ describe('WidgetFrame (with widget that has config)', () => {
   });
 
   it('renders the API menu item', () => {
-    expect(component.find('.WidgetFrame__api-btn').exists()).toBe(true);
+    expect(component.find('.WidgetFrame__api-btn').exists()).toBe(false);
```

Comment:
think makes more sense to remove the tests about the ”API" and "Save as new widget" buttons